docs/guides/getting-started.md

```
@@ -0,0 +1,106 @@
+npx supabase start
+npx supabase db push --local
+```
+
+This spins up the local Supabase stack (Postgres, Auth, Storage, Realtime, Studio) using Docker and applies our migrations from `supabase/migrations/`.
+
+You can open Supabase Studio at:
+
+- `http://127.0.0.1:54323`
+
+> **Note:** The `supabase/` directory in the repo is the single source of truth for the database schema and seed data. Never edit tables manually in Studio without also adding a migration.
+
```

markdownlint-cli2 (0.21.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)